sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -1208,27 +1196,6 @@ void analyzeActionsWithMultiSameKindActionsRunner(
                     new RecognizeLinkedEntitiesAction())
                 .setAnalyzeSentimentActions(new AnalyzeSentimentAction().setActionName(CUSTOM_ACTION_NAME),
                     new AnalyzeSentimentAction())
-                .setExtractSummaryActions(new ExtractSummaryAction().setActionName(CUSTOM_ACTION_NAME),
```

Comment:
Just double-checking: Did you intend to remove only what was commented out? Or did you intend to remove .setExtractSummaryActions too?

Reply:
Try to keep a small set of actions.